tests/testthat/test-markdown.R

```diff
@@ -648,3 +648,19 @@ test_that("can override default options", {
   ")[[1]]
   expect_match(out$get_section("description")$value, "###", fixed = TRUE)
 })
+
+test_that("image formats work", {
+  local_roxy_meta_set("restrict_image_formats", TRUE)
+
+  expect_snapshot(
+    roc_proc_text(rd_roclet(), "
+      #' Title
+      #'
+      #' ![](example.svg \"Plot title 1\")
+      #' ![](example.pdf \"Plot title 2\")
+      #' ![](example.PNG \"Plot title 3\")
+      #' @md
+      foo <- function() { }
+    ")
```

Comment:
```suggestion
    ")[[1]]
```

Will need to re-run snapshots.